src/helpers/classNames/__tests__/classNames.test.js

```diff
@@ -20,9 +20,15 @@ describe('classNames', () => {
     expect(result).toEqual('class-1 class-2 class-3');
   });
 
-  it('returns undefined if all class names are filtered out', () => {
+  it('returns empty string if called with no params', () => {
     const result = classNames();
 
-    expect(result).toEqual(undefined);
+    expect(result).toEqual('');
+  });
+
+  it('returns empty string if all params are removed', () => {
```

Comment:
`... if all params are removed` ... I am still not sure about the existence of this test. `... if all params are removed` does not make sense. I know what do you want to test but if I am user seeing this for the first time, I would be confused and consider this bug. What params are removed? How are removed? By any way, this is not related to function test.

If you want to still test this, better describe the function like `returns empty string if called with only falsy params`

Reply:
done